backend/src/schema/index.js

```diff
@@ -20,7 +20,6 @@ export default makeAugmentedSchema({
         'FILED',
         'REVIEWED',
         'Report',
```

Comment:
Why is parts in CAPS here?

Reply:
I guess that those queries in caps are related to relations instead of nodes … ?